components/script/dom/htmltableelement.rs

```diff
@@ -309,18 +309,19 @@ impl HTMLTableElementMethods<crate::DomTypeHolder> for HTMLTableElement {
                         element.local_name() == &local_name!("tbody") &&
                         element.upcast::<Node>().GetParentNode().as_deref() == Some(root)
                 },
+                CanGc::note(),
             )
         })
     }
 
     // https://html.spec.whatwg.org/multipage/#dom-table-createtbody
-    fn CreateTBody(&self, can_gc: CanGc) -> DomRoot<HTMLTableSectionElement> {
+    fn CreateTBody(&self) -> DomRoot<HTMLTableSectionElement> {
         let tbody = HTMLTableSectionElement::new(
             local_name!("tbody"),
             None,
             &self.owner_document(),
             None,
-            can_gc,
+            CanGc::note(),
```

Comment:
Why are you removing existing `CanGc` annotations? This function does not need to be changed at all.